packages/core/src/schema/index.ts

```diff
@@ -1,5 +1,8 @@
 export * from './db-schema'
 
+// Legacy models
+export * from './legacyModels/datasetsV1'
```

Comment:
I kept old datasets table alive so we can access data for now

Reply:
good